apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/SamhsaMatcherFromClaimTransformerTest.java

```diff
@@ -90,90 +88,55 @@ public void setup() {
+    ExplanationOfBenefit hhaEob = claimTransformerInterface.transform(getClaim(HHAClaim.class));
     String hhaClaimType = TransformerUtils.getClaimType(hhaEob).toString();
 
+    claimTransformerInterface =
+        new HospiceClaimTransformer(new MetricRegistry(), new NPIOrgLookup());
     ExplanationOfBenefit hospiceEob =
-        HospiceClaimTransformer.transform(
-            new TransformerContext(
-                new MetricRegistry(),
-                Optional.empty(),
-                FdaDrugCodeDisplayLookup.createDrugCodeLookupForTesting(),
-                new NPIOrgLookup()),
-            getClaim(HospiceClaim.class));
+        claimTransformerInterface.transform(getClaim(HospiceClaim.class));
     String hospiceClaimType = TransformerUtils.getClaimType(hospiceEob).toString();
 
-    ExplanationOfBenefit snfEob =
-        SNFClaimTransformer.transform(
-            new TransformerContext(
-                new MetricRegistry(),
-                Optional.empty(),
-                FdaDrugCodeDisplayLookup.createDrugCodeLookupForTesting(),
-                new NPIOrgLookup()),
-            getClaim(SNFClaim.class));
+    claimTransformerInterface = new SNFClaimTransformer(new MetricRegistry(), new NPIOrgLookup());
+    ExplanationOfBenefit snfEob = claimTransformerInterface.transform(getClaim(SNFClaim.class));
     String snfClaimType = TransformerUtils.getClaimType(snfEob).toString();
 
+    claimTransformerInterface =
+        new CarrierClaimTransformer(
+            new MetricRegistry(),
+            FdaDrugCodeDisplayLookup.createDrugCodeLookupForTesting(),
+            new NPIOrgLookup());
     ExplanationOfBenefit carrierEob =
-        CarrierClaimTransformer.transform(
-            new TransformerContext(
-                new MetricRegistry(),
-                Optional.empty(),
-                FdaDrugCodeDisplayLookup.createDrugCodeLookupForTesting(),
-                new NPIOrgLookup()),
-            getClaim(CarrierClaim.class));
+        claimTransformerInterface.transform(getClaim(CarrierClaim.class), false);
     String carrierClaimType = TransformerUtils.getClaimType(carrierEob).toString();
 
-    ExplanationOfBenefit pdeEob =
-        PartDEventTransformer.transform(
-            new TransformerContext(
-                new MetricRegistry(),
-                Optional.empty(),
-                FdaDrugCodeDisplayLookup.createDrugCodeLookupForTesting(),
-                new NPIOrgLookup()),
-            getClaim(PartDEvent.class));
-
+    claimTransformerInterface =
+        new PartDEventTransformer(
+            new MetricRegistry(), FdaDrugCodeDisplayLookup.createDrugCodeLookupForTesting());
+    ExplanationOfBenefit pdeEob = claimTransformerInterface.transform(getClaim(PartDEvent.class));
     String pdeClaimType = TransformerUtils.getClaimType(pdeEob).toString();
+
+    // TODO - check why inpatient and outpatient aren't included here
```

Comment:
Should we followup on this? (if its a followup ticket, lets create and link it here)

Reply:
no followup; this was something to remind me to go lookup/learn more about what claims are tied to SAMHSA. TODO removed.